### PR TITLE
Make index page for local development at localhost:4000 

### PR DESCRIPTION
**Pull request by CorneliaKelinske**
#412 

Is this what you had in mind?

### CODE REVIEW

**Review by alanvardy — CHANGES_REQUESTED** [review]
Thank you! I looks great.

Would you be able to `git rebase` all the extraneous commits into a single one?

image

File: lib/test_web/templates/page/index.html.heex [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<h1> Welcome to Exzeitable!</h1>
+
+<p> Here are the examplary (exzei-)tables: </p>
+<li><span><%= link "Posts", to: Routes.post_path(@conn, :index) %></span></li> 
+<li><span><%= link "Users", to: Routes.user_path(@conn, :index) %></span></li>
```
Comment (alanvardy) [inline lib/test_web/templates/page/index.html.heex:5]: Do these need to go in a `<ul>` tag?

File: lib/test_web/router.ex [hunk 2]
```diff
@@ -18,7 +18,7 @@ defmodule TestWeb.Router do
   scope "/", TestWeb do
     pipe_through(:browser)
 
-    get("/", PostController, :index)
+    get("/", PageController, :index)
```
Comment (alanvardy) [inline lib/test_web/router.ex:21]: :+1:

**Review by alanvardy — APPROVED** [review]
One more question, otherwise good to go!

File: lib/test_web/templates/page/index.html.heex (hunk 1 above)
Comment (CorneliaKelinske) [reply lib/test_web/templates/page/index.html.heex:5]: It works this way. But it might not be THE way.
So I will add ul tag

**Comment by alanvardy** [comment]
Thank you @CorneliaKelinske !

**Pull request merged**